### PR TITLE
[FIX] mrp: hide start button if Mo is cancelled

### PR DESCRIPTION
**Pull request by nouraellm**
- Hide start button if the MO is cancelled

Task ID: 2443977

### CODE REVIEW

**Comment by nouraellm** [comment]
@Whenrow

File: addons/mrp/views/mrp_workorder_views.xml [hunk 1]
```diff
@@ -84,7 +84,7 @@
                   attrs="{'invisible': [('production_state','=', 'draft')], 'readonly': [('is_user_working', '=', True)]}"/>
                 <field name="state" widget="badge" decoration-success="state == 'done'" decoration-info="state not in ('done', 'cancel')" attrs="{'invisible': [('production_state', 'in', ('draft', 'done'))]}"/>
                 <button name="button_start" type="object" string="Start" class="btn-success"
-                  attrs="{'invisible': ['|', '|', '|', ('production_state','in', ('draft', 'done')), ('working_state', '=', 'blocked'), ('state', '=', 'done'), ('is_user_working', '!=', False)]}"/>
+                  attrs="{'invisible': ['|', '|', '|', ('production_state','in', ('draft', 'done', 'cancel')), ('working_state', '=', 'blocked'), ('state', '=', 'done'), ('is_user_working', '!=', False)]}"/>
```
Comment (Whenrow) [inline addons/mrp/views/mrp_workorder_views.xml:87]: I would hide 'Pause' and 'Done' button too. What do you think ?

Comment (nouraellm) [reply addons/mrp/views/mrp_workorder_views.xml:87]: Nice catch!

**Comment by Whenrow** [comment]
robodoo r+

**Pull request closed without merging**